doc/user-guide/06-layout-configuration.adoc

```diff
@@ -131,6 +131,17 @@ multipath disks. Typically, they are part of the SAN disaster recovery
 strategy. However, there can be cases where you want to recover them. The
 information is retained in +disklayout.conf+.
 
+Some filesystems are excluded from the layout file by default if they
+are mounted under a certain path. This behavior is controlled by the
```

Comment:
I am not a native English speaker
but I am wondering
if "mounted under a certain path"
or "mounted below a certain path"
is correct English and what makes it more clear
that AUTOEXCLUDE_PATH+=( /mymountpoint )
does not exclude what is mounted diectly at /mymountpoint
but what is mounted at a directory below /mymountpoint ?
Perhaps "mounted underneath a certain path"
could make that obvious?

Reply:
Regardless of the particular wording
the explicit description
```
Note that if one of the paths is itself a mountpoint,
the filesystem is not excluded. So, if /media is a
mounted filesystem, it will not be excluded, but
if we mount the /dev/mapper/backup-backup filesystem
on /media/backup, it will get excluded, as the mountpoint
is under /media.
```
makes it very clear how it behaves.

Reply:
@jsmeix I have changed the text according to my suggestion above, and I plan to merge the PR soon today.